login.sls

```diff
@@ -5,7 +5,8 @@ include:
   - docker
 
 install_pip_for_custom_python_modules:
-  pkg.installed: [python3-pip]
```

Comment:
This is in fact wrong. You can either go with the `name` attribute - like you did - or use this:

```yaml
pkg.installed:
  - pkgs: [python3-pip]
```

Both should be fine, but the latter is a bit more streamlined with (my) other formulas. Mind chaning it? 

Thanks for spotting this!

Reply:
I picked the `name` version since this only installs a a single package, and a singleton array feels kind of strange. Moreover, I got the impression that the `pkgs` key is designed to install specific package versions (rather than latest):

```yaml
my_state:
  pkg.installed:
    - pkgs:
      - foo: 1.2.3
      - bar: 4.5.6
```
(and if you omit the versions on all entries you can of course shorten it back to a yaml array)

Based on this, do you still want the array version?